Backpack-SwiftUI/NavigationTab/Classes/BPKNavigationTab.swift

```diff
@@ -0,0 +1,80 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct BPKNavigationTab: View {
+    
+    let text: String
+    let icon: BPKIcon?
+    let selected: Bool
+    let style: BPKNavigationTabStyle
+    let onClick: () -> Void
+    
+    public init(
+        _ text: String,
+        icon: BPKIcon? = nil,
+        selected: Bool = false,
+        style: BPKNavigationTabStyle = .default,
+        onClick: @escaping () -> Void
+    ) {
+        self.text = text
+        self.icon = icon
+        self.selected = selected
+        self.style = style
+        self.onClick = onClick
+    }
+    
+    public var body: some View {
+        Button(action: onClick) {
```

Comment:
Can you refactor this to just use a BPKChip? it looks the exact same except for the corner radious, which you can make the current BPKChipStyle struct receive that size as a parameter so you dont need to duplicate that file (or this one for that matter)

Reply:
no, lets NOT do this.